### PR TITLE
Shadows now cant be picked up

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
no longer are you able to pick up shadows

## Changelog
:cl:
/:cl:

### CODE REVIEW

File: code/game/objects/effects/mines.dm [hunk 1]
```diff
@@ -101,6 +101,16 @@
 /obj/item/weapon/spider_shadow_trap/Crossed(AM as mob|obj)
 	Bumped(AM)
 
+/obj/item/weapon/spider_shadow_trap/attack_hand(mob/user as mob)
+	if(ishuman(M))
```
Comment (KathyRyals) [inline code/game/objects/effects/mines.dm:105]: Should be early returned.

File: code/game/objects/effects/mines.dm [hunk 2]
```diff
@@ -101,6 +101,16 @@
...
+		for(var/mob/O in viewers(world.view, src.loc))
```
Comment (KathyRyals) [inline code/game/objects/effects/mines.dm:106]: Probably should check for null at some point on O to avoid potential runtimes.

**Review by KathyRyals — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/mines.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/game/objects/effects/mines.dm:105]: if it returns then it dosnt trigger.

File: code/game/objects/effects/mines.dm [hunk 3]
```diff
@@ -102,14 +102,7 @@
...
 /obj/item/weapon/spider_shadow_trap/attack_hand(mob/user as mob)
-	if(ishuman(M))
-		for(var/mob/O in viewers(world.view, src.loc))
-			M.visible_message(
-				SPAN_DANGER("A gutteral screeching roar is heard right before [M] is knocked down by a huge spider leaping from above!"),
-				SPAN_DANGER("You hear a gutteral screeching roar right before something huge falling from above knocks you down!")
-			)
-		triggered = 1
-		call(src,triggerproc)(M)
+	return
```
Comment (KathyRyals) [inline code/game/objects/effects/mines.dm:105]: Did you forget to add the whole code back ?

Comment (Trilbyspaceclone) [reply code/game/objects/effects/mines.dm:105]: Nope, they no longer can be picked up, ive reised you can abuse this by triggering in in cases of being in a wall

Comment (ThePainkiller) [reply code/game/objects/effects/mines.dm:105]: I still don't get how you can pick them up when the anchored flag is set to true lmao, still, ingenious

**Review by KathyRyals — APPROVED** [review]

**Pull request merged**